UoaWDCC/younite-web
# 104 Convert Chairperson Image to Text

## Context
- Originally the the arrows and fun facts on the members pages pointing to the chairperson images were images
- Fixed it so that the arrow is an image and the fun fact itself is modifiable text

Closes #104 

## What Changed?
- Edited the arrow images to crop out the existing fun fact text
- Implemented fun fact text from strapi onto the page
- Aligned the image and text to point at the chairperson images

## How To Review
- Head over to the members page and check that the arrows pointing to the chairperson image are images, and the fun fact text is text
- Check that the fun fact text is modifiable through strapi
- Checkout next/components/members/Chairman.tsx to review the changes
